src/index.js

```diff
@@ -1,23 +1,38 @@
 import './index.css';
-import { TodoListItems } from './todolist.js';
+import * as todoList from './todolist.js';
+import * as addRemove from './add-remove.js';
+import * as storage from './storage.js';
 
-const tasks = [
-  {
-    description: 'Write article',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Go shop',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Complete 2 coding challenges',
-    completed: false,
-    index: 2,
-  },
-];
+const tasks = storage.getTasksFromStorage();
 
-TodoListItems(tasks);
-// document.body.appendChild(component());
\ No newline at end of file
+todoList.TodoListItems(tasks);
+addRemove.TodoListDelete(tasks);
+const addTask = document.querySelector('.text-area');
+addTask.addEventListener('keydown', (e) => {
+  if (e.keyCode === 13) {
+    const task = {
+      description: addTask.value,
+      completed: false,
+      index: tasks.length,
+    };
+    addRemove.TodoListAdd(tasks, task);
+    addTask.value = null;
+  }
+});
+
+const editTask = document.querySelectorAll('.item');
+editTask.forEach((i, index) => {
+  i.addEventListener('keydown', (e) => {
+    if (e.keyCode === 13) {
+      const task = {
+        description: i.value,
+        completed: false,
+        index,
+      };
+      tasks[index] = task;
+      todoList.TodoListItems(tasks);
+      storage.setTasksToStorage(tasks);
+      addRemove.TodoListDelete(tasks);
+    }
+  });
+});
```

Comment:
- [ ] Kindly make sure once you have edited the task's description to also update data in the local storage to make your project behavior professional and as required for the project. For now, you can edit tasks' descriptions but without updating data in the local storage.